src/constants.ts

```diff
@@ -1,5 +1,127 @@
+import path from 'path';
+
+/**
+ * The name of this project. Used to exclude this repo as a lintable project.
+ */
+const THIS_PROJECT_NAME = 'module-lint';
+
+/**
+ * The root directory of this project.
+ */
+const THIS_PROJECT_DIRECTORY_PATH = path.resolve(__dirname, '..');
+
 /**
  * The number of milliseconds in an hour, used to determine when to pull the
  * latest changes for previously cached repositories.
  */
 export const ONE_HOUR = 60 * 60 * 1000;
+
+/**
+ * Wherever the tool was run.
+ */
+export const WORKING_DIRECTORY_PATH = process.cwd();
+
+/**
+ * The usage text printed when the user requests help or provides invalid input.
+ */
+export const USAGE_TEXT = `
+Analyzes one or more repos for divergence from a template repo.
+
+${THIS_PROJECT_NAME} OPTIONS [ARGUMENTS...]
```

Comment:
Should this be

```suggestion
${THIS_PROJECT_NAME}
```

Since there are no options or arguments? Or maybe

```suggestion
${THIS_PROJECT_NAME} [--help]
```

Reply:
There aren't any options, but there are arguments. I've clarified what kind of arguments they should be and removed the options: [1aed3b2](https://github.com/MetaMask/module-lint/pull/33/commits/1aed3b22c1ea9e594b5603c1d3d0d880ab3dd4c5)